planetlabs/planet-client-python
# Refactor CLI

What changed:
- Rename CLI folder from scripts to cli
- Break CLI into modules: main, orders, auth, cmds, and io
- Break up CLI testing into three modules: test_cli_main, test_cli_auth, and test_cli_orders.

What didn't change:
- Functionality of the cli
- Test functionality

Small side effects:
- remove hidden coverage file (why #359 didn't remove this file, I do not know)
- update copyright of affected files to planet labs pbc.
- pin pytest-asyncio version to avoid deprecation warnings (chose to pin vs updating because if we move to aiohttp then we will no longer use pytest-asyncio)
- Add PlanetException root exception to exceptions module
- Move AuthException over to exceptions module

Part of #362
